src/external-types/reveal.js/index.d.ts

```diff
@@ -4,9 +4,199 @@
+    // - false:  No auto-sliding, even if data-autoslide is present
+    autoSlide?: 0 | 1 | false, // 1:
+    // Use this method for navigation when auto-sliding (defaults to navigateNext)
+    /* autoSlideMethod: null, // ToDo: I think this is not needed */
+    // Stop auto-sliding after user input
+    autoSlideStoppable?: boolean,
+    // Transition style for full page slide backgrounds
+    backgroundTransition?: Transition,
+    // Vertical centering of slides
+    center?: boolean,
+    // Display presentation control arrows
+    controls?: boolean,
+    // Visibility rule for backwards navigation arrows
+    controlsBackArrows?: ControlsBackArrows,
+    // Determines where controls appear
+    controlsLayout?: ControlsLayout,
+    // Help the user learn the controls by providing hints, for example by
+    // bouncing the down arrow when they first encounter a vertical slide
+    controlsTutorial?: boolean,
+    // Specify the average time in seconds that you think you will spend
+    // presenting each slide. This is used to show a pacing timer in the
+    // speaker view
+    defaultTiming?: number | null,
+    /* disableLayout?: boolean, // ToDo: I think this is not needed */
+    // The display mode that will be used to show slides
+    display?: Property.Display,
+    // Flags if the presentation is running in an embedded mode,
+    // i.e. contained within a limited portion of the screen
+    embedded?: boolean,
+    // Focuses body when page changes visibility to ensure keyboard shortcuts work
+    focusBodyOnPageVisibilityChange?: boolean,
+    // Flags whether to include the current fragment in the URL,
+    // so that reloading brings you to the same fragment position
+    /* fragmentInURL?: boolean, // ToDo: I think this is not needed */
+    // Turns fragments on and off globally
+    fragments?: boolean,
+    // Add the current slide number to the URL hash so that reloading the
+    // page/copying the URL will return you to the same slide
+    /* hash?: boolean, // ToDo: I think this is not needed */
+    // Use 1 based indexing for # links to match slide number (default is zero
+    // based)
+    /* hashOneBasedIndex?: boolean, // ToDo: I think this is not needed */
+    // Flags if we should show a help overlay when the question-mark
+    // key is pressed
+    help?: boolean,
+    // Time before the cursor is hidden (in ms)
+    hideCursorTime?: number,
+    // Hide cursor if inactive
+    hideInactiveCursor?: boolean,
+    // Push each slide change to the browser history.  Implies `hash: true`
+    /* history?: boolean, // ToDo: I think this is not needed */
+    // Enable keyboard shortcuts for navigation
+    keyboard?: boolean,
+    // Optional function that blocks keyboard events when retuning false
+    //
+    // If you set this to 'foucsed', we will only capture keyboard events
+    // for embdedded decks when they are in focus
+    /* keyboardCondition: null, // ToDo: I think this is not needed */
```

Comment:
> Optional function that blocks keyboard events when retuning false
>
> If you set this to 'foucsed', we will only capture keyboard events
> for embdedded decks when they are in focus

Again this enables custom JS functions in the frontmatter. Something we don't want, do we?

Reply:
Remove.